I 03-17 20:29:56 controller.py:59] DAG:
I 03-17 20:29:56 controller.py:59] [Task<name=compute-text-vectors>(run='# Initialize and dow...')
I 03-17 20:29:56 controller.py:59]   resources: AWS(cpus=4, {'L4': 1})]
I 03-17 20:29:56 controller.py:195] Submitted managed job 479 (task: 0, name: 'compute-text-vectors'); SKYPILOT_TASK_ID: sky-managed-2025-03-17-20-29-56-374383_compute-text-vectors_479-0
I 03-17 20:29:56 controller.py:199] Started monitoring.
I 03-17 20:29:56 state.py:451] Launching the spot cluster...
I 03-17 20:29:56 storage.py:1058] Storage type StoreType.S3 already exists.
<sky-payload>"<rich_init>[bold cyan]Optimizing[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Optimizing[/]</rich_start>"</sky-payload>
I 03-17 20:29:57 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:29:57 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 20:29:57 optimizer.py:749] 
I 03-17 20:29:57 optimizer.py:884] Considered resources (1 node):
I 03-17 20:29:57 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:29:57 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:29:57 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:29:57 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-east-1     0.80       [32m   âœ”[0m     
I 03-17 20:29:57 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Launching[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:29:57 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1a,us-east-1b,us-east-1c,us-east-1d).
[2mW 03-17 20:30:00 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:30:01 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:30:39 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:30:40 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-east-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:30:40 cloud_vm_ray_backend.py:2097] 
W 03-17 20:30:40 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:30:40 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:30:40 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 20:30:40 optimizer.py:749] 
I 03-17 20:30:40 optimizer.py:884] Considered resources (1 node):
I 03-17 20:30:40 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:30:40 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:30:40 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:30:40 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-east-2     0.80       [32m   âœ”[0m     
I 03-17 20:30:40 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:30:40 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2a,us-east-2b,us-east-2c).
[2mW 03-17 20:30:43 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 128 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:30:43 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 128 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:30:58 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 128 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:30:58 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-east-2 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:30:58 cloud_vm_ray_backend.py:2097] 
W 03-17 20:30:58 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:30:59 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:30:59 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 20:30:59 optimizer.py:749] 
I 03-17 20:30:59 optimizer.py:884] Considered resources (1 node):
I 03-17 20:30:59 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:30:59 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:30:59 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:30:59 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-west-2     0.80       [32m   âœ”[0m     
I 03-17 20:30:59 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:30:59 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2a,us-west-2b,us-west-2c,us-west-2d).
[2mW 03-17 20:31:18 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:31:52 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:32:00 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:33:27 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:33:28 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-west-2 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:33:28 cloud_vm_ray_backend.py:2097] 
W 03-17 20:33:28 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:33:28 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:33:28 optimizer.py:749] [1mEstimated cost: [0m$0.9 / hour
I 03-17 20:33:28 optimizer.py:749] 
I 03-17 20:33:28 optimizer.py:884] Considered resources (1 node):
I 03-17 20:33:28 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:33:28 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:33:28 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:33:28 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-north-1    0.85       [32m   âœ”[0m     
I 03-17 20:33:28 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:33:28 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1a,eu-north-1b).
[2mW 03-17 20:33:40 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:34:21 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:34:22 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-north-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:34:22 cloud_vm_ray_backend.py:2097] 
W 03-17 20:34:22 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:34:22 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:34:22 optimizer.py:749] [1mEstimated cost: [0m$0.9 / hour
I 03-17 20:34:22 optimizer.py:749] 
I 03-17 20:34:22 optimizer.py:884] Considered resources (1 node):
I 03-17 20:34:22 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 20:34:22 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE    COST ($)   CHOSEN   
I 03-17 20:34:22 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 20:34:22 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ca-central-1   0.89       [32m   âœ”[0m     
I 03-17 20:34:22 optimizer.py:952] -----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:34:22 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1a,ca-central-1b).
[2mW 03-17 20:34:25 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:34:26 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:34:27 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ca-central-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:34:27 cloud_vm_ray_backend.py:2097] 
W 03-17 20:34:27 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:34:27 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:34:27 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 20:34:27 optimizer.py:749] 
I 03-17 20:34:27 optimizer.py:884] Considered resources (1 node):
I 03-17 20:34:27 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:34:27 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:34:27 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:34:27 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-south-1    0.97       [32m   âœ”[0m     
I 03-17 20:34:27 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:34:27 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1a,ap-south-1b).
[2mW 03-17 20:34:36 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-south-1a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-south-1b.[0m
[2mW 03-17 20:34:38 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-south-1b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-south-1a.[0m
W 03-17 20:34:39 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ap-south-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:34:39 cloud_vm_ray_backend.py:2097] 
W 03-17 20:34:39 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:34:39 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:34:39 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 20:34:39 optimizer.py:749] 
I 03-17 20:34:39 optimizer.py:884] Considered resources (1 node):
I 03-17 20:34:39 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 20:34:39 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE      COST ($)   CHOSEN   
I 03-17 20:34:39 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 20:34:39 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-northeast-2   0.99       [32m   âœ”[0m     
I 03-17 20:34:39 optimizer.py:952] -------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:34:40 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2a,ap-northeast-2d).
[2mW 03-17 20:34:48 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-northeast-2a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-northeast-2d.[0m
[2mW 03-17 20:34:51 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-northeast-2d). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-northeast-2a.[0m
W 03-17 20:34:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ap-northeast-2 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:34:52 cloud_vm_ray_backend.py:2097] 
W 03-17 20:34:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:34:52 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:34:52 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 20:34:52 optimizer.py:749] 
I 03-17 20:34:52 optimizer.py:884] Considered resources (1 node):
I 03-17 20:34:52 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 20:34:52 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE    COST ($)   CHOSEN   
I 03-17 20:34:52 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 20:34:52 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-central-1   1.01       [32m   âœ”[0m     
I 03-17 20:34:52 optimizer.py:952] -----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:34:52 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-central-1[0m (eu-central-1a,eu-central-1b).
[2mW 03-17 20:34:58 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:35:00 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:35:01 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-central-1 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:35:01 cloud_vm_ray_backend.py:2097] 
W 03-17 20:35:01 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:35:01 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:35:01 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 20:35:01 optimizer.py:749] 
I 03-17 20:35:01 optimizer.py:884] Considered resources (1 node):
I 03-17 20:35:01 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:35:01 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:35:01 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:35:01 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-west-2     1.02       [32m   âœ”[0m     
I 03-17 20:35:01 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:35:01 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2a,eu-west-2b).
[2mW 03-17 20:35:06 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
[2mW 03-17 20:35:14 instance.py:243] create_instances: Attempt failed with An error occurred (VcpuLimitExceeded) when calling the RunInstances operation: You have requested more vCPU capacity than your current vCPU limit of 64 allows for the instance bucket that the specified instance type belongs to. Please visit http://aws.amazon.com/contact-us/ec2-request to request an adjustment to this limit.[0m
W 03-17 20:35:15 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-west-2 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:35:15 cloud_vm_ray_backend.py:2097] 
W 03-17 20:35:15 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:35:15 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:35:15 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 20:35:15 optimizer.py:749] 
I 03-17 20:35:15 optimizer.py:884] Considered resources (1 node):
I 03-17 20:35:15 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:35:15 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 20:35:15 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 20:35:15 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-west-3     1.02       [32m   âœ”[0m     
I 03-17 20:35:15 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:35:15 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3b,eu-west-3c).
[2mW 03-17 20:35:19 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (eu-west-3b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing eu-west-3c.[0m
[2mW 03-17 20:35:21 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (eu-west-3c). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing eu-west-3b.[0m
W 03-17 20:35:22 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-west-3 for {AWS(cpus=4, {'L4': 1})}. 
W 03-17 20:35:22 cloud_vm_ray_backend.py:2097] 
W 03-17 20:35:22 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:35:22 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 20:35:22 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 20:35:22 optimizer.py:749] 
I 03-17 20:35:22 optimizer.py:884] Considered resources (1 node):
I 03-17 20:35:22 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 20:35:22 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE      COST ($)   CHOSEN   
I 03-17 20:35:22 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 20:35:22 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-southeast-2   1.05       [32m   âœ”[0m     
I 03-17 20:35:22 optimizer.py:952] -------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:35:22 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-southeast-2[0m (ap-southeast-2a,ap-southeast-2c).
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Waiting for SSH access[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 20:35:50 provisioner.py:451] [2mâ””â”€â”€ [0m[2mInstance is up.[0m
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (1/3 - initializing)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (2/3 - dependencies)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (3/3 - runtime)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:36:20 provisioner.py:632] [0m[32mâœ“ Cluster launched: compute-text-vectors-479.[0m  [2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/provision.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
I 03-17 20:36:20 execution.py:332] [0mâš™ï¸Ž Syncing files.
<sky-payload>"<rich_init>[bold cyan]Syncing files[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing files[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Syncing files[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing file mounts[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/file_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:36:21 backend_utils.py:1315]   [2mSyncing (to 1 node): s3://skypilot-filemounts-root-726f1974-9aeq2zf3/job-9aeq2zf3/workdir -> ~/sky_workdir[0m
I 03-17 20:36:24 cloud_vm_ray_backend.py:4801] [0m[32mâœ“ Synced file_mounts.[0m  [2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/file_mounts.log[0m
<sky-payload>"<rich_update>[bold cyan]Mounting 1 storage[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/storage_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 20:36:24 backend_utils.py:1315]   [2mMounting (to 1 node): s3://sky-text-embeddings-multi-cluster-final -> /output[0m
I 03-17 20:36:28 cloud_vm_ray_backend.py:4892] [0m[32mâœ“ Storage mounted.[0m  [2mView logs: sky api logs -l sky-2025-03-17-20-29-57-111152/storage_mounts.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Running setup[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Running setup[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Running setup[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
I 03-17 20:36:32 cloud_vm_ray_backend.py:3357] [0m[32mâœ“ Setup detached.[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Submitting job[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Submitting job[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Submitting job[/]</rich_start>"</sky-payload>
I 03-17 20:36:35 cloud_vm_ray_backend.py:3460] [0mâš™ï¸Ž Job submitted, ID: 1
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>

[?25hI 03-17 20:36:35 recovery_strategy.py:289] Managed job cluster launched.
I 03-17 20:36:39 utils.py:131] === Checking the job status... ===
I 03-17 20:36:40 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:36:40 utils.py:140] ==================================
I 03-17 20:36:41 state.py:471] Job started.
I 03-17 20:37:02 utils.py:131] === Checking the job status... ===
I 03-17 20:37:03 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:37:03 utils.py:140] ==================================
I 03-17 20:37:23 utils.py:131] === Checking the job status... ===
I 03-17 20:37:24 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:37:24 utils.py:140] ==================================
I 03-17 20:37:44 utils.py:131] === Checking the job status... ===
I 03-17 20:37:45 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:37:45 utils.py:140] ==================================
I 03-17 20:38:05 utils.py:131] === Checking the job status... ===
I 03-17 20:38:06 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:38:06 utils.py:140] ==================================
I 03-17 20:38:26 utils.py:131] === Checking the job status... ===
I 03-17 20:38:27 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:38:27 utils.py:140] ==================================
I 03-17 20:38:47 utils.py:131] === Checking the job status... ===
I 03-17 20:38:48 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 20:38:48 utils.py:140] ==================================
I 03-17 20:39:09 utils.py:131] === Checking the job status... ===
I 03-17 20:39:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:39:10 utils.py:140] ==================================
I 03-17 20:39:30 utils.py:131] === Checking the job status... ===
I 03-17 20:39:31 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:39:31 utils.py:140] ==================================
I 03-17 20:39:51 utils.py:131] === Checking the job status... ===
I 03-17 20:39:53 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:39:53 utils.py:140] ==================================
I 03-17 20:40:13 utils.py:131] === Checking the job status... ===
I 03-17 20:40:14 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:40:14 utils.py:140] ==================================
I 03-17 20:40:34 utils.py:131] === Checking the job status... ===
I 03-17 20:40:35 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:40:35 utils.py:140] ==================================
I 03-17 20:40:56 utils.py:131] === Checking the job status... ===
I 03-17 20:40:57 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:40:57 utils.py:140] ==================================
I 03-17 20:41:17 utils.py:131] === Checking the job status... ===
I 03-17 20:41:18 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:41:18 utils.py:140] ==================================
I 03-17 20:41:38 utils.py:131] === Checking the job status... ===
I 03-17 20:41:39 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:41:39 utils.py:140] ==================================
I 03-17 20:41:59 utils.py:131] === Checking the job status... ===
I 03-17 20:42:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:42:00 utils.py:140] ==================================
I 03-17 20:42:20 utils.py:131] === Checking the job status... ===
I 03-17 20:42:22 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:42:22 utils.py:140] ==================================
I 03-17 20:42:42 utils.py:131] === Checking the job status... ===
I 03-17 20:42:46 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:42:46 utils.py:140] ==================================
I 03-17 20:43:06 utils.py:131] === Checking the job status... ===
I 03-17 20:43:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:43:10 utils.py:140] ==================================
I 03-17 20:43:30 utils.py:131] === Checking the job status... ===
I 03-17 20:43:31 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:43:31 utils.py:140] ==================================
I 03-17 20:43:51 utils.py:131] === Checking the job status... ===
I 03-17 20:43:52 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:43:52 utils.py:140] ==================================
I 03-17 20:44:12 utils.py:131] === Checking the job status... ===
I 03-17 20:44:14 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:44:14 utils.py:140] ==================================
I 03-17 20:44:34 utils.py:131] === Checking the job status... ===
I 03-17 20:44:35 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:44:35 utils.py:140] ==================================
I 03-17 20:44:55 utils.py:131] === Checking the job status... ===
I 03-17 20:44:56 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:44:56 utils.py:140] ==================================
I 03-17 20:45:16 utils.py:131] === Checking the job status... ===
I 03-17 20:45:17 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:45:17 utils.py:140] ==================================
I 03-17 20:45:37 utils.py:131] === Checking the job status... ===
I 03-17 20:45:38 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:45:38 utils.py:140] ==================================
I 03-17 20:45:58 utils.py:131] === Checking the job status... ===
I 03-17 20:45:59 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:45:59 utils.py:140] ==================================
I 03-17 20:46:19 utils.py:131] === Checking the job status... ===
I 03-17 20:46:20 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:46:20 utils.py:140] ==================================
I 03-17 20:46:40 utils.py:131] === Checking the job status... ===
I 03-17 20:46:41 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:46:41 utils.py:140] ==================================
I 03-17 20:47:01 utils.py:131] === Checking the job status... ===
I 03-17 20:47:02 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:47:02 utils.py:140] ==================================
I 03-17 20:47:23 utils.py:131] === Checking the job status... ===
I 03-17 20:47:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:47:24 utils.py:140] ==================================
I 03-17 20:47:44 utils.py:131] === Checking the job status... ===
I 03-17 20:47:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:47:45 utils.py:140] ==================================
I 03-17 20:48:05 utils.py:131] === Checking the job status... ===
I 03-17 20:48:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:48:06 utils.py:140] ==================================
I 03-17 20:48:26 utils.py:131] === Checking the job status... ===
I 03-17 20:48:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:48:27 utils.py:140] ==================================
I 03-17 20:48:47 utils.py:131] === Checking the job status... ===
I 03-17 20:48:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:48:48 utils.py:140] ==================================
I 03-17 20:49:08 utils.py:131] === Checking the job status... ===
I 03-17 20:49:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 20:49:09 utils.py:140] ==================================
I 03-17 20:49:29 utils.py:131] === Checking the job status... ===
I 03-17 20:49:30 utils.py:137] Job status: JobStatus.SUCCEEDED
I 03-17 20:49:30 utils.py:140] ==================================
I 03-17 20:49:31 state.py:561] Job succeeded.
I 03-17 20:49:31 controller.py:238] Managed job 479 (task: 0) SUCCEEDED. Cleaning up the cluster compute-text-vectors-479.
E 03-17 20:49:34 controller_utils.py:376] Failed to find the logs for the user program at ~/sky_logs/managed_jobs/sky-2025-03-17-20-29-57-111152/run.log.
I 03-17 20:49:34 controller.py:117] 
I 03-17 20:49:34 controller.py:117] == End of logs (ID: 479) ==
I 03-17 20:49:43 state.py:656] Cancellation skipped, job is already terminal
I 03-17 20:49:43 state.py:677] Cancellation skipped, job is not CANCELLING
I 03-17 20:49:45 controller.py:552] Killing controller process 269597.
I 03-17 20:49:45 controller.py:560] Controller process 269597 killed.
I 03-17 20:49:45 controller.py:562] Cleaning up any cluster for job 479.
I 03-17 20:49:46 storage.py:699] Verifying bucket for storage skypilot-filemounts-root-726f1974-9aeq2zf3
<sky-payload>"<rich_init>[bold cyan]Deleting S3 bucket [green]skypilot-filemounts-root-726f1974-9aeq2zf3[/][/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Deleting S3 bucket [green]skypilot-filemounts-root-726f1974-9aeq2zf3[/][/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Deleting S3 bucket [green]skypilot-filemounts-root-726f1974-9aeq2zf3[/][/]</rich_start>"</sky-payload>
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
I 03-17 20:49:47 storage.py:1505] [32mDeleted S3 bucket skypilot-filemounts-root-726f1974-9aeq2zf3.[0m
I 03-17 20:49:47 controller.py:571] Cluster of managed job 479 has been cleaned up.
